src/System.Private.Xml/tests/XmlSerializer/ReflectionOnly/System.Xml.XmlSerializer.ReflectionOnly.Tests.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <DefineConstants>$(DefineConstants);ReflectionOnly</DefineConstants>
     <ProjectGuid>{4050F1D1-1DD2-4B48-A17B-E3F955518C4B}</ProjectGuid>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>  
   </PropertyGroup>
```

Comment:
I removed this last one and the entries from the project files. There are still a couple of #if uapaot within `C:\git\corefx\src\System.Runtime.Serialization.Xml\tests` that I'm going to leave as I don't have context.